### PR TITLE
RHINENG-5154 fixed browser title for systems page

### PR DESCRIPTION
**Pull request by PreetiW**
## PR Title :boom:

Fixes: https://issues.redhat.com/browse/RHINENG-5154

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here.

## Documentation requires update? :memo:

- [ ] Yes
- [x] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [x] Bugfix
- [ ] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -60,6 +60,11 @@ class App extends Component {
 
     }
 
+    componentDidUpdate() {
```
Comment (r14chandra) [inline src/App.js:63]: We already have `componentDidUpdate` at line 41 and we are using `updateDocumentTitle`.

**Review by r14chandra — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (r14chandra) [reply src/App.js:63]: [#L41-L52](https://github.com/RedHatInsights/ros-frontend/blob/main/src/App.js#L41-L52)

Comment (kgaikwad) [reply src/App.js:63]: @PreetiW  -  any updates here? Since long time, it is in open state.

**Comment by r14chandra** [comment]
I am closing the PR as there has been no update. A new PR has been created to address the issue - #393.

**Pull request closed without merging**